src/FacebookUtils.js

```diff
@@ -58,8 +58,6 @@ const provider = {
           newOptions[key] = initOptions[key];
         }
       }
-      newOptions.authResponse = authResponse;
```

Comment:
Thanks for the @durunvo. I am also seeing this issue. In Facebook's docs I can't see what passing this to `FB.init` would have been for, but maybe someone who knows more about this can explain why.

If you do remove this, perhaps there is no need for the whole `authResponse` object on line 50 and line 72 could be changed to `existingResponse.userID !== authData.id`?

Reply:
FB.init require just 4 keys
```
{
  appId      : '{your-app-id}',
  status     : true,
  xfbml      : true,
  version    : 'v2.7' // or v2.6, v2.5, v2.4, v2.3
}
```
So yeah, I also think that it is not needed. But I want some explanation first just to make sure that this is a right thing to do.

But anyway I will push another commit according to your suggestion.